jmxexporter-prometheus-grafana/assets/prometheus/jmx-exporter/kafka_connect.yml

```diff
@@ -84,14 +84,14 @@ rules:
     labels:
       connector: "$1"
       task: "$2"
-  # confluent.replicator:type=confluent-replicator-task-metrics,*
-  - pattern: "confluent.replicator<type=confluent-replicator-task-metrics,(.+)=(.+),(.+)=(.+),(.+)=(.+),(.+)=(.+)><>([^:]+)"
-    name: confluent_replicator_confluent_replicator_task_metrics_$9
+  # confluent.replicator:type=confluent-replicator-task-metrics,confluent-replicator-task-topic-partition-*
+  - pattern: "confluent.replicator<type=confluent-replicator-task-metrics,(.+)=(.+),(.+)=(.+),(.+)=(.+),(.+)=(.+)><>confluent-replicator-task-topic-partition-([^:]+)"
```

Comment:
Would this make better sense to add to the blacklist/whitelist instead of lmiting them from the pattern parser ?
That way , the folks who would wanna reduce the verbosity can reduce it from there.

Reply:
sure will move that to the whitelist